README.md

```diff
@@ -17,11 +17,12 @@ steps:
     - uses: pdm-project/setup-pdm@v3
       name: Setup PDM
       with:
-        python-version: 3.9  # Version range or exact version of a Python version to use, the same as actions/setup-python
-        architecture: x64    # The target architecture (x86, x64) of the Python interpreter. the same as actions/setup-python
-        version: 1.4.0       # The version of PDM to install. Leave it as empty to use the latest version from PyPI, or 'head' to use the latest version from GitHub
-        prerelease: true     # Allow prerelease versions to be installed
-        enable-pep582: true  # Enable PEP 582 package loading globally
+        python-version: 3.9             # Version range or exact version of a Python version to use, the same as actions/setup-python
+        architecture: x64               # The target architecture (x86, x64) of the Python interpreter. the same as actions/setup-python
+        version: 1.4.0                  # The version of PDM to install. Leave it as empty to use the latest version from PyPI, or 'head' to use the latest version from GitHub
+        prerelease: true                # Allow prerelease versions of PDM to be installed
+        enable-pep582: true             # Enable PEP 582 package loading globally
+        allow-python-prereleases: true  # Allow prerelease versions of Python to be installed. For example if only 3.12-dev is available, 3.12 will fallback to 3.12-dev
```

Comment:
It's a bit long, I can shorten it to `python-prerelease`, let me know.

Reply:
The long name is okay. Let's change the value to correctly reflect the defaults, to avoid confusion.

Reply:
I guess all booleans are false by default? Should I update `prerelease` too then?

Reply:
I've done just that, and update PDM's version as well. Didn't touch Python (3.9), lmk if I should set it to 3.11.